src/Modules/Server.cpp

```diff
@@ -1027,6 +1032,13 @@ bool Server::Init() {
 		}
 	}
 
+	// there are some weird differences between the linux and windows binaries
```

Comment:
These differences are to be expected -- the game is built with totally different compilers on different platforms, so there's no reason the generated code will look the same at all! (This comment should be removed for that reason.)

Also, have you checked all supported mods? The pattern may be different on some or all of them.

Reply:
Yeah, well the differences are big enough that Linux seems to use a different entity ("viewmodel" vs "predicted_viewmodel"), so I am not even 100% confident in the signature which is mostly why I left that comment there.